cluster/config-defaults.yaml

```diff
@@ -58,17 +58,15 @@ skipper_ingress_min_replicas: "2"
 {{end}}
 skipper_ingress_cpu: "1000m"
 skipper_ingress_memory: "1Gi"
-{{if eq .Cluster.Environment "e2e"}}
-enable_dedicate_nodepool_skipper: "false"
-{{else}}
+# skipper node-pool
 enable_dedicate_nodepool_skipper: "true"
-{{end}}
-skipper_suppress_route_update_logs: "true"
+skipper_attach_only_to_skipper_node_pool: "true"
```

Comment:
This should be false for e2e now, and then we can enable it in a follow up PR, then the migration to skipper pool should be successful.